test/controllers/api/v2/sync_plans_controller_test.rb

```diff
@@ -120,6 +126,22 @@ def test_create_protected
       end
     end
 
+    def test_show
+      get :show, params: { :id => @sync_plan.id }
+
+      assert_response :success
+      assert_template 'api/v2/sync_plans/show'
+    end
+
+    def test_show_protected_specific_instance
+      allowed_perms = [{:name => @view_permission, :search => "name=\"#{@sync_plan.name}\"" }]
+      denied_perms = [{:name => @view_permission, :search => "name=\"#{@sync_plan1.name}\"" }]
+
+      assert_protected_object(:show, allowed_perms, denied_perms) do
+        get :show, params: { :id => @sync_plan.id }
+      end
+    end
+
```

Comment:
Can't seem to get the assert_protected_object test to work with denied perms. On the UI and in API I see a 404, however in the test it gives a 204.  :confused:

Reply:
That is because
```
      @sync_plan = katello_sync_plans(:sync_plan_hourly)
    ........

      @sync_plan1 = katello_sync_plans(:sync_plan_hourly)
```
both point to the same thing